### DISCUSSION TITLE
Wallet stuck and wont sync ver 1.15

### DISCUSSION THREAD

**Discussion by mikebrad070883** [Chia-Network/chia-blockchain#4726] [Support]
Hi,

Can someone please? I'm new to Chia and my wallet wont sync and gets stuck on a certain block. I have port forwarded and using a static IP address. Not sure what else to try also didnt work in V1.14 Windows10

Many thanks

**Comment by furkan16** [comment]
hi @mikebrad070883 
I think I found a solution like this can be tried by reinstalling the 1.1.5 file.
Can you also check your own sync nodes for me? It seems I have 3-4 wallet connections, how many do you have?

**Comment by mikebrad070883** [comment]
Hi

I only have 1 wallet. It seems to have stopped because it was syncing faster than the node. If I reinstall would the node need to re sync?

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Reply by furkan16** [reply]
hi
my sync resumed where it left off
thank you @mikebrad070883

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
